core/probe.cpp

```diff
@@ -416,8 +416,13 @@ void Probe::delayedInit()
     // The applicationName might be translated, so let's go with the application file base name
     m_server->setKey(QFileInfo(qApp->applicationFilePath()).completeBaseName());
     m_server->setPid(qApp->applicationPid());
-    m_server->listen();
-    ProbeSettings::sendServerAddress(m_server->externalAddress());
+    bool serverStarted = m_server->listen();
+    if (serverStarted)
+    {
+        ProbeSettings::sendServerAddress(m_server->externalAddress());
+    } else {
+        ProbeSettings::sendServerLaunchError("The address is not available");
```

Comment:
Maybe send the QTcpServer error string here, instead the hardcoded error message?